### ISSUE TITLE
Naming conventions

### ISSUE THREAD

**Issue by redahaq** [fac19/week2-guardians#46]
TLDR: Avoid using names that already have meanings in JS/ HTML etc. Produces ambiguity and reduces readability.

I'd avoid using the 'Get' bit of getElementName as variable names for selecting DOM elements as it makes your code less readable and a little confusing... you're not actually 'getting' anything here, that's happening in the Fetch request and is the name of the method you're using as well. 

What you're doing here is selecting the elements in the DOM where you'll be rendering things you 'get' from your API calls.

You could make it as simple as using
  `const title = document.querySelector(".section-movie-db__title");`   etc

This'll make it a lot easier to read. 

![image](https://user-images.githubusercontent.com/39345463/76587744-d35ea780-64dc-11ea-83b5-fa7dcf9b3913.png)

Another bit I found a little confusing was naming your reddit section with author and time details as 'footer' - maybe call it something like 'comment source' 'comment details' 'comment attribution' , whatever.. keep it specific and meaningful. Footer makes things vague and ambiguous as `<Footer>` is a semantic HTML tag.
